src/lib/libast/vmalloc/meson.build

```diff
@@ -7,3 +7,96 @@ libast_files += [
     'vmalloc/vmsegment.c', 'vmalloc/vmset.c', 'vmalloc/vmstat.c',
     'vmalloc/vmstrdup.c', 'vmalloc/vmtrace.c', 'vmalloc/vmuserdata.c'
 ]
+
+
+if cc.has_function('mallinfo', prefix: '#include <malloc.h>')
+    libast_c_args += [ '-D_lib_mallinfo=1' ]
```

Comment:
There isn't any need for the `=1` portion since without it that is the default value assigned to the symbol. I point this out because elsewhere (e.g., `-D_lib_utimensat`) we don't explicitly include the `=1`. We should either consistently set the value or omit it to use the equivalent default.